include/ismrmrd/xml.h

```diff
@@ -319,7 +320,19 @@ namespace ISMRMRD
   };
 
   struct IsmrmrdHeader
+  : public Entity
```

Comment:
This is wrong. This ismrmrd header is the deserialized version of the XML header. Not the actual acquisitions. When the XML header is an "entity" on the wire, we need a different class for it or something. We should discuss.